components/linkedin/actions/create-text-post-user/create-text-post-user.mjs

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+  key: "linkedin_create-text-post-user",
+  name: "Create a Text Post (User)",
+  description: "Create shares on LinkedIn using text, URLs, and images. [See the docs](https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/share-on-linkedin#creating-a-share-on-linkedin) for more information",
```

Comment:
Appreciate the link to docs

Reply:
Since this action specifically posts only text, you should remove references to images in the description. Can you also pass a URL as text here, or is that shared in a different way via the API?

Reply:
Yes, the **shareMediaCategory** can be one of:

1. NONE - The share does not contain any media, and will only consist of text.
2. ARTICLE - The share contains a URL.
3. IMAGE - The Share contains an image.

So I think I will remove "and images" from the description.